HungryWolfs/HomeScreen/HomeScreenViewController.swift

```diff
@@ -0,0 +1,180 @@
+//
+//  HomeScreenViewController.swift
+//  HungryWolfs
+//
+//  Created by Wolfpack Digital on 10.08.2021.
+//
+
+import UIKit
+
+class HomeScreenViewController: UIViewController {
+
+    @IBOutlet weak var searchView: UIView!
+    @IBOutlet weak var categoriesCollection: UICollectionView!
+    @IBOutlet weak var mealsCollection: UICollectionView!
+    
+    
+    
+    @IBOutlet weak var searchField: UITextField!
+    
+    private let viewModelFood: FoodViewModel = FoodViewModel()
+    private let viewModelMeal: MealsViewModel = MealsViewModel()
+    
+    var selectedIndexPath = IndexPath (item: 0, section: 0)
+    
+    override func viewDidLoad()
+    {
+        super.viewDidLoad()
+        
+        updateCornerRadius()
+        
+        categoriesCollection.delegate = self
+        categoriesCollection.dataSource = self
+        
+        mealsCollection.delegate = self
+        mealsCollection.dataSource = self
+        
+        viewModelFood.getFoods(completion: { [weak self] in
+            self?.categoriesCollection.reloadData()
+            self?.viewModelMeal.getMeals(categories: "Beef", completion: { [weak self] in self?.mealsCollection.reloadData()
```

Comment:
You can use here the first element of the Category array in FoodViewModel, instead of "Beef" (because later maybe Beef won't be the first category anymore).